Basic-Car-Maintenance/Shared/Settings/Views/ContributorsListView.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  ContributorsListView.swift
+//  Basic-Car-Maintenance
+//
+//  Created by Yashraj jadhav on 01/10/23.
+//
+
+import SwiftUI
+
+struct ContributorsListView: View {
+    
+    @StateObject var viewModel: SettingsViewModel
+    
+    var body: some View {
+        List {
+            if (viewModel.contributors?.isEmpty) != nil {
+                ForEach(viewModel.contributors!, id: \.self) { contributor in
+                    
+                    Link(destination: (URL(string: contributor.htmlURL) ?? viewModel.urls["brain-marksRepo"])!) {
+                        ContributorsProfileViews(name: contributor.login, url: contributor.avatarURL)
+                    }
+                    
+                }
+            }
+            else {
+                ForEach(viewModel.staticContributors, id: \.self) { contributor in
```

Comment:
Don't use this `staticContributors` if the data is loading, show a `ProgressView()`